assets/queries/ansible/aws/ecs_task_definition_container_has_password/metadata.json

```diff
@@ -1,9 +1,9 @@
 {
```

Comment:
Didn't change the name of the folder? Same applies in Terraform.

Reply:
Done